connectors/src/api/webhooks/slack/created_channel.ts

```diff
@@ -35,11 +35,13 @@ export function isChannelCreatedEvent(
 export interface OnChannelCreationInterface {
   event: ChannelCreatedEvent;
   logger: Logger;
+  context?: "slack_bot" | "slack";
 }
 
 export async function onChannelCreation({
   event,
   logger,
+  context = "slack",
```

Comment:
I think we want to use `provider` not context since this is a provider really and context is just a new word that we can avoid.